build/deps.js

```diff
@@ -55,7 +53,10 @@ var deps = {
 
 	DrawUI: {
 		src: [
-			'draw/DrawToolbar.js'
+			'draw/DrawToolbar.js',
+			'draw/control/Draw.Cancel.js',
+			'draw/control/Draw.RemoveLastPoint.js',
```

Comment:
The names of the similar edit control files are in the format `edit/control/Edit.Control.Edit.js`.

Would these two draw controls be better named and have more consistency if they were `draw/control/Draw.Control.Cancel.js` and `draw/control/Draw.Control.RemoveLastPoint.js`?